index.html

```diff
@@ -17,13 +17,25 @@
         <nav class="navbar">
             <div class="navbar-container">
                 <a class="navbar-brand" href="/">My Logo</a>
-                <button class="navbar-btn">
-                    <i class="fa-solid fa-bars"></i>
+                <button id="navbar-btn" class="navbar-btn">
+                    <i id="icon-btn" class="fa-solid fa-bars"></i>
                 </button>
-                <ul class="none-style nav-list">
-                    <li>Portfolio</li>
-                    <li>About</li>
-                    <li>Contact</li>
+                <ul id="nav-list" class="none-style nav-list">
```

Comment:
- [ ] Kindly add the blur effect in the background of the menu in order to match the Figma design.
### Hint:
- You can use the CSS property `backdrop-filter`.